templates/glanceapi/config/glance-api-config.json

```diff
@@ -27,7 +27,7 @@
       },
       {
         "source": "/var/lib/config-data/merged/ceph.client.openstack.keyring",
-        "dest": "/etc/ceph/ceph.client.openstack.keyring",
+        "dest": "/etc/ceph/ceph.client.{{ .CephUser }}.keyring",
```

Comment:
Nice, so provided that a given client is configured on the external ceph cluster, we can use this kind of approach to render the right file in the right location.